src/openms/include/OpenMS/ANALYSIS/OPENSWATH/MRMFeatureFilter.h

```diff
@@ -94,13 +95,81 @@ namespace OpenMS
       const TargetedExperiment & transitions);
 
     /**
-      @brief Converts a FeatureMap to a qcMLFile::Attachment
+      @brief Flags or filters features and subordinates in a FeatureMap based on a user
+        defined set of filter values derived from calling `EstimatePercRSD`.  The user supplied
+        filter_criteria represents the bounds on acceptable %RSD values.
+        NOTE that flagging nor filtering will be done on the labels and transitions type counts.
 
       @param features FeatureMap to flag or filter
-      @param attachment qcML Attachment
+      @param filter_criteria MRMFeatureQC class defining QC parameters defining the acceptable limits of the %RSD
+        where %RSD = (value std dev)/(value mean)*100%
+      @param filter_values MRMFeatureQC class filled with bounds representing the %RSD found in e.g., pooled QC samples or replicate Unknown samples
 
     */
-    void FeatureMapToAttachment(FeatureMap& features, QcMLFile::Attachment& attachment);
+    void FilterFeatureMapPercRSD(FeatureMap& features, const MRMFeatureQC& filter_criteria, const MRMFeatureQC& filter_values);
+
+    /**
+      @brief Flags or filters features and subordinates in a FeatureMap based on a user
+        defined set of filter values derived from calling `EstimateBackgroundInterferences`.  The user supplied
+        filter_criteria represents the bounds on acceptable %BackgroundInterference values.
+        NOTE that filtering is only done on the `Intensity` member.
+
+
+      @param features FeatureMap to flag or filter
+      @param filter_criteria MRMFeatureQC class defining QC parameters defining the acceptable limits of %BackgroundInterference
+        where %BackgroundInterference = (value Sample)/(value Blank)*100%
+      @param filter_values MRMFeatureQC class filled with bounds representing the average values found in e.g., pooled QC samples or replicate Unknown samples
+
+    */
+    void FilterFeatureMapBackgroundInterference(FeatureMap& features, const MRMFeatureQC& filter_criteria, const MRMFeatureQC& filter_values);
+
+    /**
+      @brief Estimate the lower and upper bound values for the MRMFeatureQC class based on a
+        user supplied template.  The template can either be initialized from the first sample 
+        (meaning all initial template values are over written) or not (meaning the initial template
+        values will be updated if the ranges are found to be too narrow)
+
+      @param[in] samples Samples (typically Standards) from which to estimate the lower and upper bound values for the MRMFeatureQC members
+      @param[in, out] filter_template A MRMFeatureQC class that will be used as a template to fill in the estimated lower and upper values.
+        A "template" is needed so that the MRMFeatureQC::meta_value_qc parameters that the FeatureMap::MetaValues that user would like estimated are known.
+      @param [in] transitions transitions from a TargetedExperiment
```

Comment:
must be `@param[in]`. The extra space will lead to `[in]` being interpreted as the variable name.
See https://github.com/OpenMS/OpenMS/blob/develop/src/openms/include/OpenMS/SYSTEM/File.h#L241for how*NOT* to do it (if you could fix that while being at it... :)).
From Doxygen: `Possible values are "[in]", "[in,out]", and "[out]"`